src/main/java/io/vertx/ext/mongo/BulkOperation.java

```diff
@@ -148,6 +127,31 @@ public static BulkOperation createUpdate(JsonObject filter, JsonObject document,
       .setMulti(multi);
   }
 
+  public CollationOptions getCollation() {
+    return collation;
+  }
+
+  public BulkOperation setCollation(CollationOptions collation) {
```

Comment:
Another way to limit the size of the PR: you could change the return type but keep both methods at the top of the file.

Reply:
Since the methods where in the middle of member declarations I thought moving them down improves readability.

I see you are kind of overwhelmed by this PR and I understand your frustration. I would suggest that we try to separate it (according to my other comments and your suggestions) into as few PRs as possible (as fiddling with it also is an effort on my side) and in the future I'll try to separate my PRs with more thought and understanding for your time.

Reply:
If changes can be broken in independent pull requests it's better.

Reply:
As it's very tedious to break it into separate PRs (since a nearly all of the changes are related to my previous introduction of various `Options` data objects to implement collation support) I'd be very thankful if you could make an exception this time, while I promise to split up my future changes into smaller chunks with higher discriminatory power. :smiling_face_with_tear: